app/views/characters/_list_section.haml

```diff
@@ -1,6 +1,7 @@
 - col_count = 6
 - col_count += 1 if local_assigns[:show_template]
 - characters = name.characters.ordered unless local_assigns[:characters]
+- characters = characters.where(retired: false) if params[:retired] == 'false'
```

Comment:
We really need some kinda shared thing for this shared config between icon and list view, but out of scope for this